### PR TITLE
Product card

### PR DESCRIPTION
**Pull request by ahmadjarafat**
-Create product card.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -2,17 +2,18 @@ import Header from "./components/header";
...
+import Home from "./pages/home";
 
 function App() {
   return (
     <BrowserRouter>
       <ThemeProvider theme={theme}>
-        <Header />
-        <Banner/>
-        <Footer/>
+        <Home />
```
Comment (Rahaf-96) [inline src/App.jsx:11]: I see here you removed the Header and Banner components, Would you please remove their imports too.

**Review by Rahaf-96 — CHANGES_REQUESTED** [review]

File: src/themes/theme.js [hunk 2]
```diff
@@ -38,6 +38,40 @@ const theme = createTheme({
   },
 });
 
+<<<<<<< HEAD
```
Comment (Rahaf-96) [inline src/themes/theme.js]: remove this line please

File: src/themes/theme.js [hunk 3]
```diff
@@ -38,6 +38,40 @@ const theme = createTheme({
...
+  color: theme.palette.lowEmphasis.main,
+  fontFamily: ["Inter", "sans-serif"].join(","),
+  [theme.breakpoints.down("sm")]: {
+    fontSize: "0.9rem",
+    fontWeight: 500,
+  },
+};
+=======
 
+>>>>>>> 2fc44e02015ba6c90516b586bbd70859fa21386a
```
Comment (Rahaf-96) [inline src/themes/theme.js]: remove this too

File: src/components/horizontal-scroll-section/styles.js [hunk 4]
```diff
@@ -1,4 +1,14 @@
+<<<<<<< HEAD
```
Comment (Rahaf-96) [inline src/components/horizontal-scroll-section/styles.js]: remove this

File: src/components/horizontal-scroll-section/styles.js [hunk 5]
```diff
@@ -1,4 +1,14 @@
...
-const StyledStack = styled()(({ theme }) => ({}));
+// TODO: fix browser compatibility issues, "make sure the scrollbar behaves the same on all browsers"
+const StyledStack = styled(Stack)(({ theme }) => ({
+  "&::-webkit-scrollbar": {
+    display: "none",
+  },
+}));
+
+export default StyledStack;
+>>>>>>> main
```
Comment (Rahaf-96) [inline src/components/horizontal-scroll-section/styles.js]: remove this please

**Review by Rahaf-96 — CHANGES_REQUESTED** [review]

File: src/components/horizontal-scroll-section/styles.js (hunk 5 above)
Comment (ahmadjarafat) [reply src/components/horizontal-scroll-section/styles.js]: Done

**Review by Rahaf-96 — APPROVED** [review]

**Pull request merged**